.taskcluster.yml

```diff
@@ -0,0 +1,36 @@
+version: 0
+tasks:
+  - provisionerId: '{{ taskcluster.docker.provisionerId }}'
+    workerType: '{{ taskcluster.docker.workerType }}'
+    extra:
+      github:
+        events:
+          - pull_request.closed
+          - pull_request.opened
+          - pull_request.reopened
+          - pull_request.synchronize
+          - pull_request.assigned
+          - pull_request.unassigned
+          - pull_request.labeled
+          - pull_request.unlabeled
+          - pull_request.edited
+          - push
+          - release
+          
+    payload:
+      maxRunTime: 3600
+      image: node
+      command:
+        - /bin/bash
+        - '--login'
+        - '-c'
+        - >-
+          git clone {{event.head.repo.url}} repo && cd repo && git config
+          advice.detachedHead false && git checkout {{event.head.sha}} && npm
+          install . && npm test
+    metadata:
+      name: Run tests for taskcluster-index in Taskcluster
+      description: write a .taskcluster.yml  that runs the tests
+      owner: '{{ event.head.user.email }}'
+      source: '{{ event.head.repo.url }}'
+allowPullRequests: collaborators
```

Comment:
There are no secrets involved in this test run, so we can allow anyone to run it -- not just collaborators.  That will eliminate the annoying "No tasks started" error that you've seen on this PR.